src/overlays/actors/ovl_En_Test3/z_en_test3.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "z_en_test3.h"
+#include "prevent_bss_reordering.h"
```

Comment:
```suggestion

#include "prevent_bss_reordering.h"
#include "z_en_test3.h"
```
Could you swap these? We try to keep this on top